### ISSUE TITLE
mapping-sheet: Extension column is blank when extensions use definitions from OCDS

### ISSUE THREAD

**Issue by duncandewhurst** [open-contracting/ocdskit#177]
After generating a mapping sheet for the OCDS for the EU profile with the following command:

```bash
ocdskit mapping-sheet release-schema.json --extension https://standard.open-contracting.org/profiles/eu/latest/en/extension.json --extension-field extension > mapping-sheet.csv
```
The extension column is only populated for 'parties/additionalContactPoints', not for its properties:

extension | section | path
-- | -- | --
OCDS for the European Union | parties | parties/additionalContactPoints
  | parties | parties/additionalContactPoints/name
  | parties | parties/additionalContactPoints/email
  | parties | parties/additionalContactPoints/telephone
  | parties | parties/additionalContactPoints/faxNumber
  | parties | parties/additionalContactPoints/url

And the same for `parties/details`:

extension | section | path
-- | -- | --
OCDS for the European Union | parties | parties/details/classifications
  | parties | parties/details/classifications/scheme
  | parties | parties/details/classifications/id
  | parties | parties/details/classifications/description
  | parties | parties/details/classifications/uri

**Comment by jpmckinney** [comment]
What version of OCDS Kit are you using? `pip list | grep ocdskit`

**Comment by jpmckinney** [comment]
Seems to be specifically in the case of arrays that reference OCDS definitions.

**Comment by duncandewhurst** [comment]
Version 1.0.1

**Comment by jpmckinney** [comment]
Fixed in 1.0.2.
